test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -161,13 +161,32 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 					util.PrintCommand(cmd)
 					out, err := cmd.CombinedOutput()
 					log.Printf("%s\n", out)
-					Expect(err).To(HaveOccurred())
+					Expect(err).NotTo(HaveOccurred())
```

Comment:
@jackfrancis was this on purpose? or a typo? If so I expect it will now fail...

Reply:
it’s on purpose, the pattern match should fail

Reply:
So yeah, this change will have to be reverted if I wasn't clear

Reply:
yup